content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,161 @@
-addresses. Now, every time that the limit is reached, an automatic reminder will be sent by
-email!
+   .. image:: in_app_purchase/send-sms.png
+      :align: center
+      :alt: The Send SMS Text Message service can be accessed from a contact's record.
 
-.. image:: in_app_purchase/image4.png
-   :align: center
+   4. Type a message in the :guilabel:`Send SMS Text Message` pop-up, then click :guilabel:`Send
+      SMS`. Odoo sends the message via SMS and log what was sent in the chatter.
+
+   The prepaid credits for the SMS service are automatically deducted from the existing pool. If
+   there are not enough credits to send the message, Odoo prompts the purchase of more.
+
+.. seealso::
+   For more information on how to use various IAP services, review the documentation below:
+
+   - :doc:`Lead mining </applications/sales/crm/acquire_leads/lead_mining>`
+   - :doc:`Enrich your contacts base with Partner Autocomplete
+     </applications/sales/crm/optimize/partner_autocomplete>`
+   - :doc:`SMS essentials </applications/marketing/sms_marketing/essentials/sms_essentials>`
+
+.. _in_app_purchase/credits:
+
+IAP credits
+===========
+
+Every time an IAP service is used, the prepaid credits for that service are spent. Odoo prompts the
+purchase of more credits when there are not enough credits to continue using the service. Email
+alerts can also be set up for when :ref:`credits are low <in_app_purchase/low-credits>`.
+
+Credits are purchased in "packs" from the `Odoo IAP website
+<https://iap.odoo.com/iap/all-in-app-services>`_, and pricing is specific to each service.
+
+.. example::
+   The `SMS service <https://iap.odoo.com/iap/in-app-services/1>`_ has four packs available, in
+   denominations of:
+
+   - **10** credits
+   - **100** credits
+   - **500** credits
+   - **1,000** credits
+
+   The number of credits consumed depends on the length of the SMS and the country of destination.
+   For more information, refer to the :doc:`SMS Pricing and FAQ
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>` documentation.
+
+.. _iap/buying_credits:
+
+Buy credits
+-----------
+
+If there are not enough credits to perform a task, the database automatically prompts the purchase
+of more credits. Users can check the current balance of credits for each service, and manually
+purchase more credits through the service's :guilabel:`Account Information` page.
+
+Manually buy credits
+~~~~~~~~~~~~~~~~~~~~
+
+   .. image:: in_app_purchase/view-services.png
+      :align: center
+      :alt: The Settings app showing the Odoo IAP heading and View My Services button.
+
+#. Go to the *Settings* application.
+#. Type `IAP` in the search bar at the top.
+#. Click :guilabel:`View My Services`.
+
+   .. image:: in_app_purchase/account-info.png
+      :align: center
+      :alt: The Account Information page for an IAP service showing the Buy Credit button.
```

Comment:
```suggestion
#. Click :guilabel:`View My Services`.

   .. image:: in_app_purchase/view-services.png
      :align: center
      :alt: The Settings app showing the Odoo IAP heading and View My Services button.

   Doing so reveals an :guilabel:`IAP Account` page, listing the various IAP services in the
   database. From here, select the desired IAP service to buy additional credits for.
   
   Clicking an IAP service from the :guilabel:`IAP Account` page reveals the :guilabel:`Account
   Information` page for that service, where additional credits can be purchased.

   .. image:: in_app_purchase/account-info.png
      :align: center
      :alt: The Account Information page for an IAP service showing the Buy Credit button.
```
some added detail to the steps, along with where I think the above image should be placed within these instructions.